src/main/java/mate/academy/service/UserService.java

```diff
@@ -14,7 +14,17 @@ public class UserService {
      * @return - user if his email is equal to passed email.
      * Return <code>null</code> if there is no suitable user
      */
+
     public User findByEmail(String email) {
+        for (int i = 0; i < users.length; i++) {
+            if (users[i].getEmail().equals(email)) {
+                return users[i];
+            }
+        }
         return null;
     }
+
+    public static User[] getUsers() {
+        return users;
+    }
```

Comment:
this method is not required by the task. instead, you should use findByEmail method
```suggestion
```